docs/content/latest/admin/cqlsh.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
Can we split into 2 separate diffs? Working with strings should go into its own diff ideally. Having different sets of changes in a single PR would slow down the review and merge process.